views/index.ejs

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title><%= config.title %></title>
+    <base href="<%= config.base %>" />
+    <!--Favicons-->
```

Comment:
the favicons are lower, we don't really need a html comment though for it

Reply:
Removed -- but I promise I didn't add the comment :)